frontend/src/scenes/experiments/Experiment.tsx

```diff
@@ -531,7 +548,7 @@ export function Experiment(): JSX.Element {
                                     trendCount={trendCount}
                                     exposure={experimentData?.parameters.recommended_running_time}
                                     sampleSize={experimentData?.parameters.recommended_sample_size}
-                                    runningTime={experimentData?.parameters.recommended_running_time}
+                                    runningTime={runningTime}
```

Comment:
this is only shown on drafts, so can use the variable, otherwise its always empty 🤷